backend/plugin/parser/mysql/test-data/test_differ_column.yaml

```diff
@@ -0,0 +1,496 @@
+  newSchema: |
+    CREATE TABLE `book` (`id` INT NULL);
+  diff: |+
+    ALTER TABLE `book` MODIFY COLUMN IF EXISTS `id` INT NULL;
+
+- oldSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) COMMENT 'Author Name' NOT NULL);
+  newSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) COMMENT 'Book Name' NOT NULL);
+  diff: |+
+    ALTER TABLE `book` MODIFY COLUMN IF EXISTS `name` VARCHAR(50) COMMENT 'Book Name' NOT NULL;
+
+- oldSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) COMMENT 'Author Name' NOT NULL);
+  newSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) COMMENT 'Book Name' NOT NULL);
+  diff: |+
+    ALTER TABLE `book` MODIFY COLUMN IF EXISTS `name` VARCHAR(50) COMMENT 'Book Name' NOT NULL;
+
+- oldSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) COMMENT 'Author Name' NOT NULL);
+  newSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) COMMENT 'AUTHOR NAME' NOT NULL);
+  diff: |+
+    ALTER TABLE `book` MODIFY COLUMN IF EXISTS `name` VARCHAR(50) COMMENT 'AUTHOR NAME' NOT NULL;
+
+- oldSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) NOT NULL);
+  newSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) COMMENT 'AUTHOR NAME' NOT NULL);
+  diff: |+
+    ALTER TABLE `book` MODIFY COLUMN IF EXISTS `name` VARCHAR(50) COMMENT 'AUTHOR NAME' NOT NULL;
+
+- oldSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) COMMENT 'AUTHOR NAME' NOT NULL);
+  newSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) NOT NULL);
+  diff: |+
+    ALTER TABLE `book` MODIFY COLUMN IF EXISTS `name` VARCHAR(50) NOT NULL;
+
+- oldSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) COMMENT 'AUTHOR NAME' NOT NULL);
+  newSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) COMMENT 'AUTHOR NAME' NOT NULL);
+  diff: ""
+
+- oldSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) NOT NULL);
+  newSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) NOT NULL);
+  diff: ""
+
+- oldSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) DEFAULT 'Harry Potter' NOT NULL);
+  newSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) NOT NULL);
+  diff: |+
+    ALTER TABLE `book` MODIFY COLUMN IF EXISTS `name` VARCHAR(50) NOT NULL;
+
+- oldSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) NOT NULL);
+  newSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) DEFAULT 'Harry Potter' NOT NULL);
+  diff: |+
+    ALTER TABLE `book` MODIFY COLUMN IF EXISTS `name` VARCHAR(50) DEFAULT 'Harry Potter' NOT NULL;
+
+- oldSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) DEFAULT 'Holmes' NOT NULL);
+  newSchema: |
+    CREATE TABLE `book` (`name` VARCHAR(50) DEFAULT 'Harry Potter' NOT NULL);
+  diff: |+
+    ALTER TABLE `book` MODIFY COLUMN IF EXISTS `name` VARCHAR(50) DEFAULT 'Harry Potter' NOT NULL;
+
+- oldSchema: |
+    CREATE TABLE `book` (`id` INT DEFAULT 0 NOT NULL);
+  newSchema: |
+    CREATE TABLE `book` (`id` INT NOT NULL);
+  diff: |+
+    ALTER TABLE `book` MODIFY COLUMN IF EXISTS `id` INT NOT NULL;
```

Comment:
ALTER TABLE `book` ALTER COLUMN `id` INT DROP DEFAULT;

Shall we have "DROP DEFAULT"?